spec/integration/subscriptions_auth_token_spec.rb

```diff
@@ -6,8 +6,12 @@
   describe "creating an auth token" do
     let(:path) { "/subscriptions/auth-token" }
     let(:address) { "test@example.com" }
-    let(:topic_id) { "business-tax-corporation-tax" }
     let(:frequency) { "daily" }
+
+    let(:topic_id) do
+      (create :subscriber_list, slug: "business-tax-corporation-tax").slug
```

Comment:
Can we use `create(:subscriber_list, slug: "business-tax-corporation-tax").slug` here as the current syntax doesn't look very familiar?

Reply:
Sure, I see where you're coming from - updated 👍.